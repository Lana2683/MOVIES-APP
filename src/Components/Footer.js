import React from 'react';

export const Footer = () => <footer/>

export default Footer;
