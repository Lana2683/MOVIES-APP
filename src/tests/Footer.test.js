import React from "react";
import { render } from '@testing-library/react';
import Footer from "../Components/Footer/Footer";

describe('Footer', () => {
    test('render Footer snapshot', () => {
        const { asFragment } = render(<Footer/>);
        expect(asFragment(<Footer/>)).toMatchSnapshot()
    })
})
