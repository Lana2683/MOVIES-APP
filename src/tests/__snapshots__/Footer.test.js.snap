// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render Footer snapshot 1`] = `
<DocumentFragment>
  <footer />
</DocumentFragment>
`;
